examples/kubernetes/demo-vol-update.yml

```diff
@@ -0,0 +1,99 @@
+apiVersion: storage.k8s.io/v1alpha1
```

Comment:
Let's change this tutorial to use v1beta1

Reply:
Done